webpack.config.js

```diff
@@ -31,7 +31,8 @@ module.exports = {
         test: /\.jsx?$/,
         include: [
           path.join(__dirname, 'client/'),
-          path.join(__dirname, 'common/')
+          path.join(__dirname, 'common/'),
+          path.join(__dirname, 'server/')
```

Comment:
@BerkeleyTrue this has solved my syntax errors for import/export from server files